src/style.css

```diff
@@ -1 +1,78 @@
-/* Styles go here */
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.image {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.nav {
+  display: flex;
+  align-self: stretch;
+}
+
+.nav-list {
+  align-self: center;
+  margin: 0;
+  padding: 0;
+  text-transform: uppercase;
+  display: flex;
+  list-style: none;
+}
+
+.nav-item {
+  align-self: stretch;
+  display: flex;
+}
+
+.nav-item:not(:last-of-type) {
+  margin-right: 20px;
+}
+
+.nav-link {
+  position: relative;
+  padding: 0;
+  color: #000;
+  text-decoration: none;
+  padding-top: 22.5px;
+  padding-bottom: 23.5px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+}
+
+.nav-link:hover {
+  color: #00acdb;
+}
+
+.is-active:link {
```

Comment:
do you need pseudo selector?

Reply:
tests do not work without pseudo selector .is-active:link
I combine his with .nav-link:hover and .is-active:visited